src/hotspot/share/code/aotCodeCache.cpp

```diff
@@ -985,6 +982,8 @@ CodeBlob* AOTCodeReader::compile_code_blob(const char* name, int entry_offset_co
 
 // ------------ process code and data --------------
 
+#define BAD_ADDRESS_ID -2
```

Comment:
Can you please add a comment to indicate why -1 is not used.
From the comment in `id_for_address`, I guess it is because -1 is a valid id for representing jump to itself in static call stub. Is that correct?
```
  int id = -1;
  if (addr == (address)-1) { // Static call stub has jump to itself
    return id;
  }
```

Reply:
Yes, it is correct. I will add the comment:
```
// Can't use -1. It is valid value for jump to iteself destination
// used by static call stub: see NativeJump::jump_destination().
```